playbooks/ansible/roles/sit.cephfs/templates/smb_share.conf.j2

```diff
@@ -13,3 +13,7 @@ read only = no
 {%- for option, value in volume.samba.options.items() | default([]) +%}
 {{ option }} = {{ value }}
 {%- endfor +%}
+
+[ceph_snapshot]
```

Comment:
Shouldn't the snapshots be available through the same share that is used to access the filesystem normally ?

In any case, I think the share name should depend on `name` and `config.be.name` at least to make sure it doesn't conflict with other shares in case the configuration includes more than one share.